### DISCUSSION TITLE
Check si on a notre énergie au max avec le small event du pet

### DISCUSSION THREAD

**Discussion by ghost** [Crownicles/Crownicles#1968] [Nouvelles idées]
Je viens de gagner de l'énergie en étant max et dans le code je vois pas de check donc peut-être j'ai mal regardé et y'a un check mal fait ou y'en a pas du tout.

**Comment by leuhvraiminoryo** [comment]
![image](https://github.com/DraftBot-A-Discord-Adventure/DraftBot/assets/50586611/e33204d0-fd0e-4826-91ed-a27dda584827)

document de support si jamais, au point du managePickedPetInteraction, dans le cas du win food y'a une condition étou mais rien de semblable pour le win energy
